### PR TITLE
chore: make spec migration screen text more generic

### LINKED ISSUE #21926: Update wording of 'rename specs' area when the specs are not actually being renamed
## **Summary**
I know this step is for renaming and moving but it feels odd to have everything say ‘rename’ when in this case there isn’t a rename and just a move:

Screen Shot 2022-05-19 at 2 07 18 PM

[https://cypressio.slack.com/archives/C02MYBT9Y5S/p1652990922196199](https://cypressio.slack.com/archives/C02MYBT9Y5S/p1652990922196199|smart-card)

**Acceptance Criteria**

If we are not renaming the spec extension (like for an existing Dashboard project)

1. Have heading say ‘Move existing specs’
1. Have sub description say ‘In this step, we’ll automatically move your existing spec files.
1. Remove the sub-heading in the content that says ‘We recommend automatically…..etc'

┆Issue is synchronized with this [Jira Task](https://cypress-io.atlassian.net/browse/UNIFY-1801) by [Unito](https://www.unito.io)
┆Attachments: <a href="https://cypress-io.atlassian.net/rest/api/2/attachment/content/12035">Screen Shot 2022-05-19 at 2.07.18 PM.png</a>
┆author: Matthew Schile
┆epic: UI/UX Improvements
┆friendlyId: UNIFY-1801
┆priority: Medium
┆sprint: Fast Follows 1
┆taskType: Task

### PR DESCRIPTION
**Pull request by lmiller1990**
- Closes #21926

### User facing changelog

N/A

### Additional details

Make the text more generic, as to cover all the cases where we automatically change specs:

- rename
- move
- rename and move

### Steps to test

Just open launchpad to any Cypress 9 project and have a look.

### How has the user experience changed?

Improved language in migration UI to be more clear about intended and recommended behavior.

## Before

image

## After

Screen Shot 2022-06-16 at 12 14 58 am

### PR Tasks

- [na] Have tests been added/updated? **It's just a json file, so no, there isn't any to update**
- [x] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only) **YES** 10.1.1
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: packages/frontend-shared/src/locales/en-US.json [hunk 1]
```diff
@@ -543,7 +543,7 @@
     "after": "After",
     "heresWhy": "here's why:",
     "renameAuto": {
-      "title": "We recommend automatically renaming your specs in this step",
+      "title": "We recommend automatically updating your specs in this step",
```
Comment (marktnoonan) [inline packages/frontend-shared/src/locales/en-US.json:546]: "updating" sounds to broad to me, what we _recommend_ is definitely renaming. This title probably doesn't need a change.

File: packages/frontend-shared/src/locales/en-US.json [hunk 2]
```diff
@@ -598,7 +598,7 @@
       "description": "Complete the steps below to migrate your project to Cypress 10",
       "step1": {
         "title": "Rename existing specs",
-        "description": "In this step, we'll automatically rename and move your existing spec files.",
+        "description": "In this step, we'll automatically change existing spec files to match the new convention.",
```
Comment (marktnoonan) [inline packages/frontend-shared/src/locales/en-US.json:601]: This also seems too broad compared to what we have now. How about:

> In this step, we'll automatically rename and/or move your existing spec files as needed.

Which keeps the scope narrow to what we will actually do, and allows it to be correct even if we are only doing one thing or the other. We could leave all other text the same I think.

File: packages/frontend-shared/src/locales/en-US.json [hunk 3]
```diff
@@ -598,7 +598,7 @@
       "description": "Complete the steps below to migrate your project to Cypress 10",
       "step1": {
         "title": "Rename existing specs",
```
Comment (marktnoonan) [inline packages/frontend-shared/src/locales/en-US.json:600]: In the screenshot, this title is changed, but it's not here?

**Review by marktnoonan — CHANGES_REQUESTED** [review]
"Updating specs" describes something potentially much broader than renaming and moving. I've suggested a smaller change that might still be generic enough.

File: packages/frontend-shared/src/locales/en-US.json (hunk 2 above)
Comment (lmiller1990) [reply packages/frontend-shared/src/locales/en-US.json:601]: I actually had some variation of this but ended up flip flopping back to "update", but happy to go with this.

File: packages/frontend-shared/src/locales/en-US.json (hunk 3 above)
Comment (lmiller1990) [reply packages/frontend-shared/src/locales/en-US.json:600]: Oh, let's use "Migrate existing specs" for the title. It's the perfect description for what we are doing, it's the migration UI, after all.

File: packages/frontend-shared/src/locales/en-US.json (hunk 1 above)
Comment (lmiller1990) [reply packages/frontend-shared/src/locales/en-US.json]: Good point, reverted

**Review by marktnoonan — APPROVED** [review]
I'm good with this, migrate is indeed a good title imo.

File: packages/frontend-shared/src/locales/en-US.json [hunk 4]
```diff
@@ -597,8 +597,8 @@
       "title": "Migration Helper",
       "description": "Complete the steps below to migrate your project to Cypress 10",
       "step1": {
-        "title": "Rename existing specs",
-        "description": "In this step, we'll automatically rename and move your existing spec files.",
+        "title": "Migrate existing specs",
+        "description": "In this step, we'll automatically rename and/or move your existing spec files as needed.",
```
Comment (rachelruderman) [inline packages/frontend-shared/src/locales/en-US.json:601]: Simpler version, up to you! 
```suggestion
        "description": "In this step, we'll automatically update your existing spec files.",
```

Comment (rachelruderman) [reply packages/frontend-shared/src/locales/en-US.json:601]: Hahaha just caught up on the whole thread on wording between you and @marktnoonan . My 2 cents is that `rename and/or move` is a bit wordy and makes me think, whereas "update" covers both and I don't have to cerebrally digest what's happening. It's a simple thing we're doing, imo the markup should be short & sweet

Comment (marktnoonan) [reply packages/frontend-shared/src/locales/en-US.json:601]: When I read "update" I feel like like it could do absolutely anything to my files. It's not clear that the changes are limited to moving/renaming, that in this step we aren't going to actually touch the internals of files (like we do other places).

Comment (rachelruderman) [reply packages/frontend-shared/src/locales/en-US.json:601]: What if it's "update your existing spec file paths" instead of "update your existing spec files"?

fwiw I'm not married to either approach and will support whatever you choose 😃

**Review by rachelruderman — APPROVED** [review]

File: packages/frontend-shared/src/locales/en-US.json (hunk 4 above)
Comment (lmiller1990) [reply packages/frontend-shared/src/locales/en-US.json:601]: I'm happy with either, just gonna ship this as is. I think they are both fine and combined with the UI, it's pretty clear what's going on.
